src/commands/build/AndroidBuilder.js

```diff
@@ -178,4 +180,21 @@ export default class AndroidBuilder extends BaseBuilder {
       }
     }
   }
+
+  async collectAndValidateCredentialsFromCI(credentialMetadata) {
+    const creds = {
+      keystorePath: process.env.EXP_ANDROID_KEYSTORE_PATH,
```

Comment:
Can we name these environment variables without the `EXP_` prefix please?

Reply:
done